doc/Type/Metamodel/PrivateMethodContainer.pod6

```diff
@@ -42,6 +42,25 @@ Adds a private method C<$code> with name C<$name>.
 
 Returns a hash of C«name => &method_object»
 
+=head2 method private_methods
+
+    method private_methods($obj)
+
+Returns a list of private method names.
+
+=head2 method private_method_names
+
+    method private_method_names($obj)
+
+Alias to C<private_methods>.
+
+=head2 method find_private_method
+
+    method find_private_method($obj, $name)
+
+Locates a private method. Otherwise, returns L<C<Mu>|/type/Mu> if it doesn't
+exist.
+
 =begin comment
 
 TODO: document find_private_method, once we've figured out how to represent
```

Comment:
This TODO should probably be deleted...

Reply:
>You should probably also establish some identifier convention, like preceding them with a _.

Is there any reason for this? I'd daresay that the convention for private methods is already part of the syntax when both declaring them and calling them. Wouldn't doing something like `!_private-method` introduce more syntactic noise?

Thanks for the great feedback, JJ. I'll close this PR and then submit them separately.

Reply:
I could say that my sight is not what it used to be, and that a single `!` does not really stand out. On the other hand, mixing `_` with `-` is downright ugly. So I don't know. You're probably right. There's also no convention as long as private attributes go. Still... Well, I really don't know.

Reply:
As an older Rakoon, I agree with JJ, the '!' really doesn't stand out well, and the leading underscore convention for private methods (and attributes) I believe is used in Perl as well as other languages.